SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -730,14 +731,11 @@ void CNEMOEulerSolver::Centered_Residual(CGeometry *geometry, CSolver **solver_c
   unsigned short iVar, jVar;
   bool err;
 
-  auto numerics = numerics_container[CONV_TERM];
+  CNumerics* numerics = numerics_container[CONV_TERM];
```

Comment:
Centered_Residual was out-of-whack, since it isnt used very often.  It seems to be working now.